src/python/ensembl/brc4/runnable/core_server.py

```diff
@@ -55,7 +55,10 @@ def connect(self) -> None:
         )
 
     def set_database(self, db_name: str) -> None:
-        self._connector.database = db_name
+        try:
+            self._connector.database = db_name
+        except:
+            logging.critical(f"Unknown database: '{db_name}' - Not located on host !")
```

Comment:
I don't think this is needed: if there is an exception here, it should be allowed to be raised and the script should stop anyway